recipies/ScallopedPotato.html

```diff
@@ -8,5 +8,26 @@
 </head>
 <body>
   <h1> Scalloped Potato </h1>
+  <img src="https://imagesvc.meredithcorp.io/v3/mm/image?url=https%3A%2F%2Fimages.media-allrecipes.com%2Fuserphotos%2F6067206.jpg&w=596&h=792&c=sc&poi=face&q=85" alt="">
+<h3>Description</h3>
+<body>
+ <p> Scalloped Potatoes are an easy classic recipe, perfect for your Easter dinner, Christmas, Thanksgiving or even just for Sunday dinner.</p>
+
+ <p>In this side dish, thinly sliced potatoes and onions are layered in an easy homemade cream sauce and baked until tender, golden, and bubbly. Potato perfection!</p> 
+</body>
+<h3>Ingredients</h3>
+<ul>
+  <li>8 large potatoes, peeled and sliced </li>
+  <li>14 slices Cheddar cheese </li>
+  <li>½ cup all-purpose flour </li>
+  <li>1 pinch salt and pepper to taste </li>
+  <li>1 cup skim milk </li>
+</ul>
+<h3>Steps</h3>
+<ol>
+  <li>Preheat oven to 350 degrees F (175 degrees C). Grease a casserole dish.</li>
+  <li>Layer the bottom the casserole dish with potatoes. Cover the potatoes with a layer of cheese, followed by a light dusting of flour, salt and pepper. Repeat the layering process until all of the ingredients are used up. Pour milk over the entire casserole. Cover with aluminum foil.</li>
+  <li>Bake in the preheated oven until the potatoes are tender, about 45 minutes. Remove foil and return to oven; bake until cheese is bubbly and beginning to brown.</li>
+</ol>
 </body>
 </html>
```

Comment:
- [ ] Incorrect indentation in whole HTML document

Refer to this website for the right indentation [https://www.learnhowtoprogram.com/introduction-to-programming-part-time-react-track/git-html-and-css/html-indentation-and-spacing](https://www.learnhowtoprogram.com/introduction-to-programming-part-time-react-track/git-html-and-css/html-indentation-and-spacing)